decidim-elections/app/models/decidim/votings/voting.rb

```diff
@@ -97,6 +99,15 @@ def cta_button_text_key
       def attachment_context
         :admin
       end
+
+      def hybrid_voting?
+        voting_type == "hybrid"
+      end
+
+      def online_voting?
+        voting_type == "online"
+      end
```

Comment:
Should we add an `in_person_voting?` method?

Reply:
Hey @leio10!
I guess you were reviewing by commit and didn't that that method has been added in a later commit? :)
BTW, I gave a quick shot at `enum` as you recommended and it was 🤩 until I tried to use it in a `select` - there for some reason it wasn't able to show the selected option (although it showed the list of options and store the correct one in db when the user selects one).
Might try to give it another spin tomorrow morning 👍

Reply:
Found some inspiration reviewing @slickepinne's PR to make the enum work in a `select` 🎉